IntelliTect.TestTools.WindowsTestWrapper/IntelliTect.TestTools.WindowsTestWrapper/DesktopControls.cs

```diff
@@ -145,6 +145,17 @@ protected T FindWinControlByControlId<T>(string controlId, Func<UITestControl, T
             return found;
         }
 
+        protected T FindControlBySibling<T>(Func<UITestControl, T> controlType, UITestControl siblingControl ) where T : UITestControl
+        {
+            if (controlType == null)
+                throw new ArgumentNullException(nameof(controlType));
+            if (siblingControl == null)
+                throw new ArgumentNullException(nameof(siblingControl));
+            T found = controlType(siblingControl);
+            found.SearchConfigurations.Add(SearchConfiguration.NextSibling);
```

Comment:
I think so. If the method is called with controlType or siblingControl being null, an exception is thrown. Beyond that, controlType is simply newing up a UiTestControl object (or derived class like WpfControl) with siblingControl as the parent.

WHICH... as I typed that, realized is a flaw in the logic. Sibling control should not be the parent. I will make some tweaks to fix this. :)